.travis.yml

```diff
@@ -83,6 +83,11 @@ jobs:
     - <<: *test-libraries
       python: 3.8
 
+    - <<: *test-pyinstaller
+      python: 3.9-dev
+    - <<: *test-libraries
+      python: 3.9-dev
```

Comment:
3.9 is officially released now, you should be able to just do `3.9`?

Reply:
You are right, just in time!